DESCRIPTION

```diff
@@ -11,6 +11,8 @@ Imports:
     epp (>= 0.4.1),
     fastmatch (>= 1.1),
     mvtnorm,
+    plyr,
```

Comment:
Where is `plyr` used? I don't have recollection of using that.

Reply:
Looks like this is only used in 1 place actually, https://github.com/mrc-ide/eppasm/blob/master/R/read-spectrum-pop1.R#L43

Reply:
hmmm, not really needed there. but not worth the effort to unwind it!